CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ do before contributing is probably sign up for the [LORIS developers'
 mailing list](http://www.bic.mni.mcgill.ca/mailman/listinfo/loris-dev).
 
 Your next step before issuing a pull request is to review our
-[Coding Standards](./docs/CodingStandards). If you are doing
+[Coding Standards](https://github.com/aces/Loris/blob/minor/docs/CodingStandards.md). If you are doing
```

Comment:
This should not include a full URL, it's just missing the extension (and according to what someone said on a different PR, shouldn't have the `./`, I think)

Reply:
Yeah I think this could just be `docs/CodingStandards.md`

Reply:
I just updated the code to the non-url style.